src/vivarium_nih_us_cvd/components/interventions.py

```diff
@@ -0,0 +1,6 @@
+from vivarium_public_health.treatment import LinearScaleUp
+
+
+class OutreachInterventionScaleUp(LinearScaleUp):
```

Comment:
Why are you setting this to None? It seems unnecessary and would enable you not to have to subclass at all.

Reply:
The LinearScaleUp defaults were causing the break so I just wipe them out.

https://github.com/ihmeuw/vivarium_public_health/blob/main/src/vivarium_public_health/treatment/scale_up.py#L142-L143